### PR TITLE
POM implementation and better selector

### PR DESCRIPTION
**Pull request by dianakrog**
POM implementation and better selector
image

### CODE REVIEW

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-    )
+    loginPage.visitLoginPage()
+    loginPage.signIn(userName, pass)
+    productList.selectTShirt()
+    itemPage.addTShirtItem()
+    shoppingCart.goShoppingCart()
+    shoppingCart.checkoutCart()
+    informationPage.checkoutUserInformation(firstName, lastName, postalCode)
+    overViewe.finishOverview()
+    checkoutComplete.checkoutOrdenComplete(messageValidateOrderComplete)
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: rename the method as `verifyOrderCompletion`

File: cypress/page/login.page.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  
+  public signIn(userName: string, password: string): void {
+    cy.get(this.userNameField).type(userName);
+    cy.get(this.passwordField).type(password);
+    cy.get(this.loginButton).click();
+    }
+}
+  
+export { LoginPage };
+
```
Comment (holgiosalos) [inline cypress/page/login.page.ts]: remove whitespaces on line 26

File: cypress/e2e/buy-shirt.cy.ts [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
+  InformationPage, OverViewPage
+} from '../page'
+
+const loginPage = new LoginPage()
+const productList = new ProductListPage()
+const shoppingCart = new ShoppingCartPage()
+const itemPage = new ItemPage()
+const checkoutComplete = new CheckoutCompletePage()
+const informationPage = new InformationPage()
+const overViewe = new OverViewPage()
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: a better name for this object can be `overviewPage`

File: cypress/page/overview.page.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+class OverViewPage {
```
Comment (holgiosalos) [inline cypress/page/overview.page.ts]: rename the class as `OverviewPage`

File: cypress/e2e/buy-shirt.cy.ts (hunk 1 above)
Comment (dianakrog) [reply cypress/e2e/buy-shirt.cy.ts]: Fixed

File: cypress/page/login.page.ts (hunk 2 above)
Comment (dianakrog) [reply cypress/page/login.page.ts]: Fixed

File: cypress/e2e/buy-shirt.cy.ts (hunk 3 above)
Comment (dianakrog) [reply cypress/e2e/buy-shirt.cy.ts]: Fixed

File: cypress/page/overview.page.ts (hunk 4 above)
Comment (dianakrog) [reply cypress/page/overview.page.ts]: Fixed

File: cypress/e2e/buy-shirt.cy.ts [hunk 5]
```diff
@@ -1,21 +1,33 @@
...
-    )
+    loginPage.visitLoginPage()
+    loginPage.signIn(userName, pass)
+    productList.selectTShirt()
+    itemPage.addTShirtItem()
+    shoppingCart.goShoppingCart()
+    shoppingCart.checkoutCart()
+    informationPage.checkoutUserInformation(firstName, lastName, postalCode)
+    overviewe.finishOverview()
+    verifyOrderCompletion.checkoutOrdenComplete(messageValidateOrderComplete)
```
Comment (holgiosalos) [inline cypress/e2e/buy-shirt.cy.ts]: in this case what you should change is the method's name, the previous name for the page object was ok. Sorry for not be clear. Suggestion: `checkout.verifyOrderCompletion(messageValidateOrderComplete)`

Comment (dianakrog) [reply cypress/e2e/buy-shirt.cy.ts]: fix suggestion, thanks

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**